wallet/src/account/output_cache/mod.rs

```diff
@@ -118,10 +118,10 @@ impl PoolData {
 }
 
 pub enum TokenCurrentSupplyState {
-    Fixed(Amount, Amount), // fixed to a certain amount
-    Lockable(Amount),      // not known in advance but can be locked once at some point in time
-    Locked(Amount),        // Locked
-    Unlimited(Amount),     // limited only by the Amount data type
+    Fixed(Amount, Amount),              // fixed to a certain amount
+    Lockable(Amount), // not known in advance but can be locked once at some point in time
+    Locked(#[allow(dead_code)] Amount), // Locked
+    Unlimited(Amount), // limited only by the Amount data type
 }
```

Comment:
The indentation of the comments is a bit broken

Reply:
Auto format...